README.md

```diff
@@ -32,6 +32,10 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: tortellini-tools/action@main
+      with:
+        curations: .tortellini/curations.yml or https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/curations.yml
```

Comment:
it is confusing to see X or Y in an example.
Can we have to separate examples:
1-using local config files
2-Fecthing config files from GitHub

Reply:
I chose to just remove getting the curations file from a URL, as this is not typical behavior. curations are expected to live next to the code in the repo